### ISSUE TITLE
Error during craft setup - Argument 1 passed to craft\services\Fields::getFieldById() must be of the type integer, null given

### ISSUE THREAD

**Issue by nettum** [engram-design/SuperTable#236]
### Description
When running `./craft setup` on a fresh database with newest versions of craft and super-table, super-table fails at the `> applying existing project config ...` step.

```
Exception 'craft\errors\MigrationException' with message 'An error occurred while executing the "craft\migrations\Install migration: Argument 1 passed to craft\services\Fields::getFieldById() must be of the type integer, null given, called in /.../vendor/verbb/super-table/src/services/SuperTableService.php on line 345'

in /.../vendor/craftcms/cms/src/db/MigrationManager.php:260
```

### Steps to reproduce

1. Take existing craft installation with production db-dump and upgrade it to newest versions of craft and super-table.
2. Verify that everything works after update (this seems fine).
3. Enable project config and generate project.yaml file.
4. Checkout the project in another folder and set it up with a empty database
5. Run `./craft setup` to verify that a clean install is possible (this will not happen as it will throw and exception on the `applying existing project config` step).

### Additional info

- Plugin version: 2.1.6
- Craft version: 3.1.7

Will investigate some more (try without `project.yaml` when installing etc.)

FYI I'm not able to test `/actions/super-table/plugin/check-content-tables` as the user table in the database are empty (this is strange to me) - but it's probably empty because the `project.yaml` sync fails on `./craft setup`.
Also FYI, when checking `/actions/super-table/plugin/check-content-tables` on the existing craft installation that has the prod db dump I get "No errors found." (whee :smile_cat:)

Any tips appreciated!

**Comment by nettum** [comment]
Did try another workaround:

1) Set `'useProjectConfigFile' => false`.
2) Run `./craft setup`.
3) Login to admin.
4) Enable `'useProjectConfigFile' => true` again.
5) Run sync on next admin pageload.

This result in the following error when config-sync comes to the following request:  
`/index.php?p=<cpTrigger>/actions/config-sync/apply-yaml-changes`:  

![image](https://user-images.githubusercontent.com/798156/52261775-b8f5ea00-292a-11e9-84e6-41e0015f89d0.png)

**Comment by engram-design** [comment]
Hmm, it certainly sounds like there's a misconfiguration in your project.yaml file. I don't suppose you've be able to share this? Also on the Craft Slack (handle `crawf`) if you need to send privately.

**Comment by nettum** [comment]
Sorry for the delay, didn't have a chance to look at it further until now!   
After upgrading to 2.1.7 and starting over it seems to import the project settings nicely from CLI when running `./craft setup` on an empty database.  Not sure what I did different, but it works :smile: 

I do get another error, but this is probably related to something else/another plugin (not really easy too see from the stacktrace):   
`Exception 'craft\errors\MigrationException' with message 'An error occurred while executing the "craft\migrations\Install migration: No primary site exists'`

Closing this issue and will get back to you if I get any other errors related to super-table.
